apps/package.json

```diff
@@ -119,6 +119,7 @@
     "messageformat": "0.2.2",
     "mkdirp": "0.5.1",
     "mochify": "2.18.0",
+    "mockfirebase": "^0.12.0",
```

Comment:
My bad - I didn't understand the "left-most non-zero digit" behavior of the caret prefix.  Looks good then!